wfl/Profiles/FoodOrderProfile.cs

```diff
@@ -25,7 +26,30 @@ protected override void Configure(EntityConfiguration<FoodOrder> configuration)
                             o => rc => rc.GetByIdAsync(o.Id)
                         )
                     )
-                 );
+                 )
+                 .UseActionTransform(actions => actions
+                    .WithName("create-menu-selection")
+                    .WithRepresentation("menu-selection")
+                    .WithMethod(ActionMethod.Create)
+                    .WithEncoding("application/x-www-form-urlencoded")
+                    .WithLink<FoodOrder, MenuSelectionsController>(o => mc => mc.CreateMenuSelectionAsync(null))
```

Comment:
Since a menu selection will always be in the context of a food order, you could force the URL to be `/api/food-order/{something}/menu-selection` and then you wouldn't need the food order id as a `text` field that people can edit.